### PR TITLE
Use TCP protocol in in_syslog plugin instead of UDP

### PR DESCRIPTION
**Pull request by abenbachir**
Replacing files for rsyslog and syslog-ng for OMS and LAD:
Replace "@" by "@@".
Replace "udp" by "tcp"

This PR should be merged after this DSC [PR#545](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/545).

### CODE REVIEW

File: installer/conf/rsyslog-lad.conf [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # OMS Syslog collection for workspace LAD (SYSLOG_PORT=%SYSLOG_PORT%)
-# To be added for each facility/min_severity: <facility>.<min_severity>     @127.0.0.1:%SYSLOG_PORT%
```
Comment (abenbachir) [inline installer/conf/rsyslog-lad.conf]: @canfikret I know this change migh not cause any conflicts with LAD, but just asking if any issue may happend ?

File: installer/conf/rsyslog-lad.conf [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
+# To be added for each facility/min_severity: <facility>.<min_severity>     @@127.0.0.1:%SYSLOG_PORT%
```
Comment (simathih) [inline installer/conf/rsyslog-lad.conf]: why do we need the extra @?

File: installer/conf/omsagent.d/syslog.conf [hunk 3]
```diff
@@ -2,7 +2,7 @@
   type syslog
   port %SYSLOG_PORT%
   bind 127.0.0.1
-  protocol_type udp
+  protocol_type tcp
```
Comment (simathih) [inline installer/conf/omsagent.d/syslog.conf]: how are you planning on verifying this change before release?

File: installer/conf/rsyslog-lad.conf (hunk 2 above)
Comment (ers-msft) [reply installer/conf/rsyslog-lad.conf]: +1

File: .gitignore [hunk 4]
```diff
@@ -9,3 +9,4 @@
 /build/x86
 /test/docker-test/*[.html, .log, .pyc]
 .vscode
+.idea
```
Comment (ers-msft) [inline .gitignore]: why?

Comment (hestolz) [reply .gitignore]: it was just part of the original commit
it is just .gitignore rule to ignore some IDE generated files i think

**Pull request merged**